.cirrus.yml

```diff
@@ -21,28 +21,24 @@ env:
     FEDORA_NETAVARK_IMAGE: "fedora-netavark-${IMAGE_SUFFIX}"
     AARDVARK_DNS_BRANCH: "main"
     AARDVARK_DNS_URL: "https://api.cirrus-ci.com/v1/artifact/github/containers/aardvark-dns/success/binary.zip?branch=${AARDVARK_DNS_BRANCH}"
+    FEDORA_NETAVARK_AARCH64_AMI_ID: "ami-02da46d8fa51e5316"
```

Comment:
OH! This is the wrong image!  [ami-02da46d8fa51e5316 is the one for podman.  ](https://cirrus-ci.com/task/5118432189349888).  I found this by looking up the image in EC2 and noticed the `Name` tag "fedora-podman-aws-arm64-c5353795591864320".  This is why you're having packaging and compiler issues.  The image you want is `ami-00d7ccde5eaacc50e` which is for `fedora-netavark-aws-arm64` :grin:

Reply:
Ugh, this is horrible.  I really need to do something to make that less confusing.  Otherwise every single `ami-*` looks just like every other `ami-*` :cry:   Why won't AWS just let us use the damn name!